### PR TITLE
Fix url of community repo where to file a PR

### PR DESCRIPTION
**Pull request by stephanme**
Link directly to https://github.com/cloudfoundry/community/blob/main/org/cloudfoundry.yml using an absolute url.

### CODE REVIEW

**Review by LeePorte — APPROVED** [review]

**Comment by emalm** [comment]
I'm not sure we want the process that the RFC defines to be so tightly tied to the structure and location of the YAML files in the community repository: whenever that changes for technical reasons, then we'll have to keep updating this RFC with the new details.

**Comment by LeePorte** [comment]
> I'm not sure we want the process that the RFC defines to be so tightly tied to the structure and location of the YAML files in the community repository: whenever that changes for technical reasons, then we'll have to keep updating this RFC with the new details.

That's a valid point. I guess it's a balance between user friendliness and frequency of updating.

**Review by loewenstein — APPROVED** [review]

File: toc/rfc/rfc-0002-github-members.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ As defined in the [roles](https://github.com/cloudfoundry/community/blob/main/to
 should be a member of the `cloudfoundry` GitHub org.
 The leads and approvers within a Working Group are best positioned to decide whether an individual has met the minimum contribution criteria to become an official contributor.
 
-A Working Group approver or lead should raise a PR on the [community](github.com/cloudfoundry/community) repository to propose adding a new member to the `cloudfoundry` GitHub org.
+A Working Group approver or lead should raise a PR on the [community](https://github.com/cloudfoundry/community/blob/main/org/cloudfoundry.yml) repository to propose adding a new member to the `cloudfoundry` GitHub org.
 This PR should add the member directly to the YAML file describing the organization membership, as automation will synchronize the GitHub org membership with the contents of this file periodically.
```
Comment (emalm) [inline toc/rfc/rfc-0002-github-members.md:27]:
```suggestion
A Working Group approver or lead should raise a [PR on the community repository](https://github.com/cloudfoundry/community/pulls) to propose adding a new member to the `cloudfoundry` GitHub org.
This PR should add the member directly to the [YAML file describing the organization membership](https://github.com/cloudfoundry/community/blob/main/org/cloudfoundry.yml), as automation will synchronize the GitHub org membership with the contents of this file periodically. 
```

**Review by emalm — CHANGES_REQUESTED** [review]
I think it'll be clearer if we link to the PR list for the community repo and then have a separate link to the existing YAML file with the org structure.

**Pull request merged**